### PR TITLE
Project details UI

### PR DESCRIPTION
**Pull request by deepsidhu85**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR adds in the latest activity, latest samples updated (both display last 10 items), and basic project information such as description, number of samples, number of automated workflow executions setup, number of members, and date created to the project details page. **Note: Previous change to have links take user to the project samples page by default when visiting a project have now been reverted back to the project details page**

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

image

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Create a project
2. Do a bunch of actions (Add/remove members and shared groups, add samples, update samples, attach files to samples)
3. Visit the `Details` page for the project
4. Verify that maximum last 10 activities and last 10 samples are displayed
5. Verify the information in the `Project Information` section are correct. Hovering over the icons will display what the number represents
6. Verify that all links that were pointing to the project samples page (as the default when visiting a project) now point back to the original details page

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -341,6 +341,17 @@ en:
...
       next: Next
       previous: Prev
+    project_dashboard:
+      activity_title: Latest Activity
+      info_title: Project Information
+      information:
+        created_on: Created on
+        no_description: No description provided for project
+        number_of_automated_workflow_executions: Number of automated workflow executions in project
+        number_of_members: Number of members in project
```
Comment (ericenns) [inline config/locales/en.yml:351]: **nitpick (blocking):** Can you remove `in project` from the next 2 translations

File: config/locales/en.yml [hunk 2]
```diff
@@ -341,6 +341,17 @@ en:
     pagination:
       next: Next
       previous: Prev
+    project_dashboard:
+      activity_title: Latest Activity
+      info_title: Project Information
+      information:
+        created_on: Created on
+        no_description: No description provided for project
+        number_of_automated_workflow_executions: Number of automated workflow executions in project
```
Comment (ericenns) [inline config/locales/en.yml:350]: **nitpick (blocking):** Can you change this to `Number of automated pipelines configured`

**Review by ericenns — CHANGES_REQUESTED** [review]
This is looking good, just a few comments.

In addition can you remove the `subtitle` from the show page as we show it under the `Project Information` sidebar.

File: app/components/project_dashboard/information_component.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <%= t(:"components.project_dashboard.information.no_description") %>
+  <% end %>
+</p>
+
+<hr class="h-px my-4 bg-gray-200 border-0 dark:bg-gray-700"/>
+
+<span class="flex items-center">
+  <%= viral_tooltip(title: t(:"components.project_dashboard.information.number_of_samples")) do %>
+    <span class="flex items-center">
+      <%= viral_icon(name: :squares_2x2, color: :subdued, classes: "h-5 w-5 m-0 mr-2") %>
```
Comment (ericenns) [inline app/components/project_dashboard/information_component.html.erb:14]: **nitpick (blocking):** Can you change this to the flask icon that is used elsewhere to denote samples.

File: config/locales/en.yml (hunk 1 above)
Comment (deepsidhu85) [reply config/locales/en.yml]: Updated in 24d7fa9977a9919ec1c5d1cb43205c9ca9af9db3

File: config/locales/en.yml (hunk 2 above)
Comment (deepsidhu85) [reply config/locales/en.yml]: Updated in 24d7fa9977a9919ec1c5d1cb43205c9ca9af9db3

File: app/components/project_dashboard/information_component.html.erb (hunk 3 above)
Comment (deepsidhu85) [reply app/components/project_dashboard/information_component.html.erb]: Fixed in 24d7fa9977a9919ec1c5d1cb43205c9ca9af9db3

**Comment by deepsidhu85** [comment]
> This is looking good, just a few comments.
> 
> In addition can you remove the `subtitle` from the show page as we show it under the `Project Information` sidebar.

Subtitle has been removed from the project details page in 24d7fa9977a9919ec1c5d1cb43205c9ca9af9db3

**Review by ericenns — APPROVED** [review]
Looks good to me!

File: app/components/activities/list_item_component.html.erb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<li class="mb-10 ms-4">
+<li class="mb-4 ms-4 activity">
```
Comment (joshsadam) [inline app/components/activities/list_item_component.html.erb:1]: **nitpick:** I don't really like random classes being added for testing purposes, is there another way you can do the test without adding this?

Comment (deepsidhu85) [reply app/components/activities/list_item_component.html.erb:1]: Going to leave for now and we can update it later

**Pull request merged**